tox_conda/plugin.py

```diff
@@ -146,7 +151,7 @@ def _run_conda_process(args, venv, action, cwd):
 
 
 @hookimpl
-def tox_testenv_create(venv, action):
+def tox_testenv_create(venv, action, _test_leave_tmp_env=False):
```

Comment:
I'm not sure that test instrumentation should come here, could it be possible to retrieve the content of the yaml by some monkey patching of `YAML` or `NamedTemporaryFile`?